infra/import_graph.py

```diff
@@ -0,0 +1,94 @@
+import networkx as nx
+
+def _type_prefix_match(prefixes, address):
+    """For a given address, find the type matching the address.
+
+    Takes an object containing an array of {prefix, type} pairs, and
```

Comment:
In Python, such pairs are more idiomatically represented by a

```python
import collections
AddressType = collections.namedtuple("AddressType", ("prefix", "type"))
```

rather than a `dict`. `dict`s are used when the keyset is arbitrary;
they are maps, not structs.

Reply:
:+1: